gatsby/pages.rst

```diff
@@ -90,63 +103,23 @@ Create a new page and link it in the index.
 
         import React from 'react'
         import { Link } from 'gatsby'
+        import Layout from '../components/layout'
 
         const PloneconfPage = () => (
-          <div>
-            <h1>Ploneconf training</h1>
-            <p>That's a page created at the training.</p>
-            <Link to="/">Go to the homepage</Link>
-          </div>
+          <Layout>
+            <div>
+              <h1>Ploneconf training</h1>
+              <p>That's a page created at the training.</p>
+              <Link to="/">Go to the homepage</Link>
+            </div>
+          </Layout>
         )
 
         export default PloneconfPage
 
 
 Components are very useful when you need to reuse a certain pattern in different pages.
-Usually components are located in a `components` folder and imported where needed.
-
-Exercise
-++++++++
-
-Create a new component that renders the page title in a common way, and use it in every page.
-
-..  admonition:: Solution
-    :class: toggle
 
-    Create `components` folder and a new file `header.js`
-
-    .. code-block:: none
+Usually components are located in a `components` folder and imported where needed (like `Layout`).
 
-        import React from 'react'
-      
-        export default Header = ({label}) => (
-          <div
-            className="header"
-            style={{
-                backgroundColor: blue;
-                color: rgb(255, 255, 255);
-                padding: 1em;
-                fontSize: 72px;
-                textAlign: center;
-            }}
-          >
-            {label}
-          </div>
-        )
-
-    Then we need to import `Header` component in our pages (index.js for example), and use it.
-
-    .. code-block:: none
-      
-        ...
-        import Header from '../components/header';
-        
-        const PloneconfPage = () => (
-          <div>
-            <Header label="Ploneconf Tokyo 2018" />
-            <h1>Welcome to Ploneconf trainings</h1>
-            <p>That's a page created at the training.</p>
-            <Link to="/">Go to the homepage</Link>
-          </div>
-        )
-    
+.. note:: In components/layout.js there is an example of custom component that add some styles and use other components.
```

Comment:
Double backticks needed.

"of a custom component that adds some styles and uses other components."